docs/pages/guides/permissions.md

```diff
@@ -58,10 +90,28 @@ Since the `android.permissions` manifest property doesn't work in the bare workf
 
 > **Note:** you have to define the `xmlns:tools` attribute on `<manifest>` before you can use the `tools:node` attribute on permissions.
 
-## Permissions on Web
+</ConfigReactNative>
+
+<ConfigClassic>
+
+By default, apps built with the Classic Build system will include **all** permissions supported by Expo. This is so that your standalone app will match its behavior in the [Expo Go][expo-go] app and simply "work out of the box" no matter what permissions you ask for, with hardly any configuration needed on your part.
```

Comment:
```suggestion
By default, apps built with the Classic Build system will include **all** permissions required by any Expo SDK package (with exceptions only for background permissions). This is so that your standalone app will match its behavior in the [Expo Go][expo-go] app and simply "work out of the box" no matter what permissions you ask for, with minimal configuration required by the developer.
```